packages/babel-cli/test/index.js

```diff
@@ -39,7 +39,9 @@ const readDir = function (loc, filter) {
 
 const saveInFiles = function (files) {
   // Place an empty .babelrc in each test so tests won't unexpectedly get to repo-level config.
-  outputFileSync(".babelrc", "{}");
+  if (!fs.existsSync(".babelrc")) {
```

Comment:
This doesn't seem needed. This line is above the `files` loop because then those will overwrite it if there is one in the test.